packages/text-input/_autosuggest.scss

```diff
@@ -57,12 +55,10 @@
             width: 100%;
             padding-right: jkl.$spacing-xl; // to accomodate button
         }
+    }
 
-        &--open {
-            > input {
-                border-radius: jkl.rem(3px) jkl.rem(3px) 0 0;
-            }
-        }
+    &[aria-expanded="true"] &__input-group > input {
```

Comment:
Er dette å anse som en breaking change i CSSen?

Reply:
Sukk, ja 😑

Reply:
Legger til den gamle håndteringen igjen for å unngå breaking. Etter å ha testet litt nå ser det ut som det to reglene catcher _hver sin_ edge case der menyen er åpen uten at tekstfeltet har skarpe hjørner nederst.